game/configuration.cc

```diff
@@ -415,6 +449,27 @@ int PaHostApiNameToHostApiTypeId (const std::string& name) {
 	throw std::runtime_error("Invalid PortAudio HostApiTypeId Specified.");
 }
 
+unsigned int LanguageToLanguageId(const std::string& name) {
+	if (name == _("Asturian")) return 1;
+	if (name == _("Danish")) return 2;
+	if (name == _("German")) return 3;
+	if (name == _("English")) return 4;
+	if (name == _("Spanish")) return 5;
```

Comment:
what are all those magic values refering to (1, 2, 3, ...)? is that some order somewhere?

Reply:
As stated below the "enum" parent needs an integer value to work correctly. The numbers here are arbitrary and doesn't refer to anything except the selected language in the parent scope of the schema.xml

Reply:
ok